pdns/pdns_recursor.cc

```diff
@@ -3451,7 +3464,7 @@ int main(int argc, char **argv)
     ::arg().set("packetcache-ttl", "maximum number of seconds to keep a cached entry in packetcache")="3600";
     ::arg().set("max-packetcache-entries", "maximum number of entries to keep in the packetcache")="500000";
     ::arg().set("packetcache-servfail-ttl", "maximum number of seconds to keep a cached servfail entry in packetcache")="60";
-    ::arg().set("server-id", "Returned when queried for 'id.server' TXT or NSID, defaults to hostname")="";
+    ::arg().set("server-id", "Returned when queried for 'id.server' TXT or NSID, defaults to hostname, set custom or 'disabled'")="";
```

Comment:
Perhaps we should default to "disabled" to prevent leaking the hostname? Especially since the behavior will change on an upgrade. Probably not a big deal though.

Reply:
This behaviour mimics the Auth. Users that change the `version-string` setting will also change this one.